model/Licensing/Classes/License.md

```diff
@@ -0,0 +1,70 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# License
+
+## Summary
+
+Abstract class for the portion of a LicenseExpression representing a license.
```

Comment:
I have a slightly different conceptual model - a "License Expression" is just text which gets parsed into a logical model which follows more of a tree structure with classes and subclasses.

You might want to take a look at the diagram in the SPDX spec repo and the text in the [SPDX RDF Terms](https://spdx.org/rdf/terms/) for a reference on the 2.X.model.  The class names should line up and you can just copy/paste the descriptions if you agree with the 2.X model.